src/main/java/duke/commands/FindCommand.java

```diff
@@ -0,0 +1,34 @@
+package duke.commands;
+
+import duke.data.TaskList;
+import duke.data.exception.DukeException;
+import duke.data.exception.EmptyDescriptionException;
+import duke.storage.Storage;
+import duke.ui.Ui;
+
+/**
+ * Encapsulates Find command's operation
+ */
+public class FindCommand extends Command {
+    private String filter;
+
+    /**
+     * Constructor for FindCommand
+     * @param rest the user input after the command
+     * @throws EmptyDescriptionException if input is empty
+     */
```

Comment:
Javadoc should have an empty line between [description and parameter section](https://se-education.org/guides/conventions/java/intermediate.html#comments). I have noticed this issue in many other files too. 
```suggestion
     * Constructor for FindCommand
     *
     * @param rest the user input after the command
     * @throws EmptyDescriptionException if input is empty
     */
```